docs/sections/Docs-Buttons/index.jsx

```diff
@@ -4,8 +4,38 @@ export default class Buttons extends React.Component {
   render() {
     return (
       <div>
-        <h1>Buttons</h1>
-        <button className="btn">Button</button>
+        <h1>Normal Buttons</h1>
+        <div className="doc-bottom-space">
+          <button className="btn" disabled>Button</button>
+          <button className="btn -info">Button</button>
+          <button className="btn -success">Button</button>
+          <button className="btn -warning">Button</button>
+          <button className="btn -danger">Button</button>
+        </div>
+        <h1>Hovered / Darker Buttons</h1>
```

Comment:
No need to show these :smile:. Might as well remove the `-#{class}-dk` *modifier*s.